tests/test_cli.py

```diff
@@ -309,6 +309,26 @@ class SampleBlock2(Block):
                 indent=2,
                 sort_keys=True)
 
+    @skipIf(not niocore_installed, 'niocore required for buildrelease')
+    def test_buildrelease_git_remote_url_parse(self):
```

Comment:
Is it time to start splitting up these test files? This is the first one where I wanted to just test one piece of the command without needing to run through the whole thing.